modyn/trainer_server/internal/trainer/maybe_measure_gpu_ops.py

```diff
@@ -0,0 +1,31 @@
+from types import TracebackType
+from typing import Optional, Type
+
+import torch
+from modyn.common.benchmark import Stopwatch
+
+
+class MaybeMeasureGPUOps:
+    def __init__(  # type: ignore[no-untyped-def]
+        self, measure: bool, measurement_name: str, device: str, stop_watch: Stopwatch, **kwargs
```

Comment:
nit: can we rename measure to enabled? that is the common name for things like this I believe, at least I have seen this pattern often

Reply:
done